### ISSUE TITLE
Package not found: Repo Tagged with 1.x

### ISSUE THREAD

**Issue by shmolf** [composer/packagist#1180]
- https://packagist.org/packages/shmolf/noted-hydrator
- https://github.com/shmolf/noted-hydrator/releases

I've tried
- `composer require shmolf/noted-hydrator`
   - Reports: `Could not find a matching version of package shmolf/noted-hydrator. Check the package spelling, your version constraint and that the package is available in a stability which matches your minimum-stability (stable).`
- `composer show shmolf/noted-hydrator --all`
   - Reports: `Package shmolf/noted-hydrator not found`

I know sometimes caches aren't updated immediately. So I've waited about 15 hours, just in case. But it seems the package is still not available.

I've tried "Googling" and looking for similar circumstances, but all I've found are instances where the publisher didn't tag their repo, or were intentionally trying to use a private/local repository.

**Comment by stof** [comment]
This is only an issue when using Composer 1. Composer 2 exposes the metadata for the new package.

@Seldaek is there a delay now in the dumping of new packages for the v1 metadata format ?

**Comment by Seldaek** [comment]
The policy announced here https://blog.packagist.com/deprecating-composer-1-support/ was put in place yesterday.. so i guess that's what's causing this.

You should really use composer 2, but if you can't then use it at least in some dummy folder to require that package and trigger one install of it.. and then it should become available on v1 within 15min if all goes well.

**Comment by stof** [comment]
oh yeah, I forgot about that new policy.

I think this needs to be added in the troubleshooting doc on the composer website, and maybe also in the warning sent by packagist.org to composer 1

**Comment by shmolf** [comment]
I'll upgrade to Composer 2. Seems like the most appropriate solution.

**Comment by shmolf** [comment]
Following [the upgrade instructions](https://blog.packagist.com/composer-2-0-is-now-available/), and myself using Ubuntu 18.04, I ran the following commands:
```bash
cd;
php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');";
# The really long command checking the hash

# and since `which composer` cmd determined it's installed @ `/usr/bin/composer`, I used the suggested options
sudo php composer-setup.php --install-dir /usr/bin --filename composer
php -r "unlink('composer-setup.php');"
```

After upgrading composer to 2.0, both commands from the issue description worked successfully.

**Comment by shmolf** [comment]
Thank you @stof and @Seldaek

**Comment by shmolf** [comment]
To be clear to those who come across this issue in the future (including myself for my other hosts), the shell commands referenced are from the [installer page](https://getcomposer.org/download/), which was linked to by the Upgrade Instructions, [towards the bottom of Section 2](https://blog.packagist.com/composer-2-0-is-now-available/#2-how-easy-is-it-to-upgrade).
